app/src/components/features/rules/RuleBuilder/Help/index.tsx

```diff
@@ -65,6 +65,19 @@ const ruleTypeToNotionDocIdMap = {
   [RuleType.USERAGENT]: "8522c44d1f1a42e1aad766847f9d4785",
 };
 
+const defaultDocEndpointsMap = {
```

Comment:
`defaultDocMockUrls`.


Also can we keep the complete urls here. Doesn't make sense to keep the endpoints only. In case we change our host of something